### DISCUSSION TITLE
MacOS light background not blurring

### DISCUSSION THREAD

**Discussion by PREPONDERANCE** [ghostty-org/ghostty#3988] [Q&A] [answered]
## Versions

```bash
MacOS = 15.0.1
Ghostty = 1.0.0
```

## Description

I've set separate themes for light mode and dark mode and set the background-blur and opacity as stated in the following configuration.

```bash
# Global theme
theme = dark:niji,light:terminal basic

# Font configuration
font-size = 14
font-family = FiraCode Nerd Font Mono

# Background configuration
background-opacity = 0.75
background-blur-radius = 20
```

The dark mode performs fine enough with the background blur but the light mode wouldn't show any blur except for the header bar, shown in the following image.

<img width="882" alt="截屏2024-12-30 11 22 15" src="https://github.com/user-attachments/assets/e051e11e-7702-4a74-8662-d27617dcdf60" />

- I've tried rebooting the ghostty app but nothing works.
- I've tried setting the background-opacity to 0. It works as complete transparent but any value beyond 0 would render completely white background.

**Comment by 00-kat** [comment, marked as answer]
Dup #3666 which was fixed by #3324.
